rosdep/python.yaml

```diff
@@ -6803,34 +6803,84 @@ python3-flake8:
     '*': ['python%{python3_pkgversion}-flake8']
     '7': null
   ubuntu: [python3-flake8]
+python3-flake8-blind-except:
+  opensuse: [python3-flake8-blind-except]
```

Comment:
For opensuse is the package name python**3**-... or python-...? https://software.opensuse.org/package/python-flake8-blind-except

Reply:
The suggestions all have the `python3-` variant. Looking at our current keybase the `python3-` variant is more common than the bare `python-` but if our style guides say otherwise I am down to change these.

```
rg -N 'opensuse.*python' | awk '/python2-/ { py2++ } /python3-/ {py3++} /python-/ { justpy++ } END {print("py2: " py2); print("py3: " py3); print("just py: " justpy);}'
py2: 45
py3: 105
just py: 35
```

Reply:
Ah, I see I might have created this ambiguity because I linked the `python-` name for the packages in my PR body. That was a typo and I've corrected it.